Reason: Fix build with ffmpeg:6
Upstream: https://github.com/cmus/cmus/issues/1251
Written by: Calvin Walton for Exherbo, based on PR comment
    https://github.com/cmus/cmus/pull/1254#issuecomment-1528063605

diff --git a/ip/ffmpeg.c b/ip/ffmpeg.c
index 09ed3fc..54b16e9 100644
--- a/ip/ffmpeg.c
+++ b/ip/ffmpeg.c
@@ -202,8 +202,10 @@ static int ffmpeg_open(struct input_plugin_data *ip_data)
 			break;
 		}
 
+#if LIBAVCODEC_VERSION_INT < AV_VERSION_INT(59, 18, 100)
 		if (codec->capabilities & AV_CODEC_CAP_TRUNCATED)
 			cc->flags |= AV_CODEC_FLAG_TRUNCATED;
+#endif
 
 		if (avcodec_open2(cc, codec, NULL) < 0) {
 			d_print("could not open codec: %d, %s\n", cc->codec_id, avcodec_get_name(cc->codec_id));
